Test_objectMover is flaky

**What steps did you take and what happened:**
[A clear and concise description on how to REPRODUCE the bug.]

The Test_objectMover is failing from time to time with the following error
```yaml
    mover_test.go:724: 
        Expected
            <time.Time>: 2022-01-10T22:00:03.398330287Z
        to be <
            <time.Time>: 2022-01-10T22:00:03.398330287Z
    --- FAIL: Test_objectMover_backupTargetObject/Cluster (0.03s)
```

Recent run:
* https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/periodic-cluster-api-test-main/1480660217195663360

/kind bug
[One or more /area label. See https://github.com/kubernetes-sigs/cluster-api/labels?q=area for the list of labels]

--- a/cmd/clusterctl/client/cluster/mover_test.go
+++ b/cmd/clusterctl/client/cluster/mover_test.go
@@ -704,7 +704,7 @@ func Test_objectMover_backupTargetObject(t *testing.T) {
 				g.Expect(string(fileContents)).To(Equal(expectedFileContents))
 
 				// Add delay so we ensure the file ModTime of updated files is different from old ones in the original files
-				time.Sleep(time.Millisecond * 5)
+				time.Sleep(time.Millisecond * 50)
 
 				// Running backupTargetObject should override any existing files since it represents a new backup
 				err = mover.backupTargetObject(node, dir)

